### ISSUE TITLE
Not merged comment for enum existing fields

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#277]
Source

```
"""
  Enum without comments
"""
enum TestEnum {
  Value1
  Value2
}


"""
  Extended Enum with comments and new Value
"""
enum TestEnum {
  """Value1"""
  Value1

  """Value2"""
  Value2

  """Value3"""
  Value3
}

```

Result
```
"""
Extended Enum with comments and new Value
"""
enum TestEnum {
  Value1
  Value2
  """
  Value3
  """
  Value3
}
```

This worked before, as far as I remember.

**Comment by ardatan** [comment]
Could you try with the following version?
`1.7.3-bdb2407.0 `

**Comment by Fi1osof** [comment]
@ardatan , yes, with this one all works fine. Thanks!

**Comment by ardatan** [comment]
Let's keep it open until we release new version :)

**Comment by ardatan** [comment]
Available in 1.7.3 !
